packages/utils/src/createSchemaUtils.ts

```diff
@@ -87,7 +87,7 @@ class SchemaUtils<
   getDefaultFormState(
     schema: S,
     formData?: T,
-    includeUndefinedValues = false
+    includeUndefinedValues: boolean | "once" = false
```

Comment:
Also, update the function docs to add details for the new option
```suggestion
    includeUndefinedValues: boolean | "noRecursion" = false
```